lib/pull_request.rb

```diff
@@ -161,23 +161,22 @@ def tell_dependency_manager_what_dependabot_is_changing
 private
 
   def ci_workflow_run_id
-    return @ci_workflow_run_id unless @ci_workflow_run_id.nil?
-
-    # No method exists for this in Octokit,
-    # so we need to make the API call manually.
-    uri = "https://api.github.com/repos/alphagov/#{@api_response.base.repo.name}/actions/runs?head_sha=#{@api_response.head.sha}"
-    ci_workflow_api_response = GitHubClient.get(uri)
+    @ci_workflow_run_id ||= begin
```

Comment:
Nice refactor 👍 typo in commit message though.

Reply:
🤦‍♂️ Good catch, thanks